lib/extensions/action_controller/base.rb

```diff
@@ -0,0 +1,66 @@
+module Extensions::ActionController::Base
+  # Gets the current layout used by this controller.
+  #
+  # @return [String] The layout used by the current controller.
+  def current_layout
+    _layout
+  end
+
+  # Gets the parent layout of the given layout, as specified in the layout hierarchy.
+  #
+  # @param layout [Symbol|nil] The layout to obtain the parent of. If this is nil, obtains the
+  #                            current controller's parent layout.
+  # @return [Symbol] The parent layout of the given layout.
+  def parent_layout(layout = nil)
+    layout ||= current_layout
```

Comment:
maybe I should go:
```ruby
controller.parent_layout
```
to give the current controller's parent layout

and

```ruby
controller.parent_layout(of_layout: 'layout')
```
what do you think?